prediction-applier-extension/src/main/resources/alfresco/module/alfresco-hxinsight-connector-prediction-applier-extension/module.properties

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+# SDK Sample module
+
+# ==== Beginning of Alfresco required/optional properties ====== #
+# NB: These properties are filtered at build time by Maven, single
+# sourcing from POM properties
+module.id=${project.artifactId}
+#module.aliases=myModule-123, my-module
+module.title=${project.name}
+module.description=${project.description}
+module.version=${project.version}
+
+# The following optional properties can be used to prevent the module from being added
+# to inappropriate versions of the WAR file.
+# module.repo.version.min=2.0
```

Comment:
What should be the minimal supported version? 7.1? Right now we are only installing new aspect to the model and it works fine with previous version (tested with 7.1), but I'm not sure how it will behave when we will add some Java code

Reply:
Let's go with 7.1 for now and raise the question with Martin in the Thursday meeting. The answer is bound to depend on the cost of supporting more versions. I think the biggest cost is running tests against them, but since we know we need to support 7.4, 23.1 and 23.2 then I doubt it will cost us much more to support 7.3, 7.2 and 7.1 too.